tools/getkeys/index.js

```diff
@@ -21,10 +17,11 @@ const { exec } = require('child_process');
                 const envName = secretName.split('-').join('__'); // secret name can't contain underscores
                 console.log(`Setting environment variable ${envName}...`);
                 setEnv(envName, response.value);
+                rc[envName] = response.value;
```

Comment:
rather than just sticking these as props off the root can we put them in a sub-object like we do with tokens, maybe call is secrets?